model/eligibility/Eligibility.php

```diff
@@ -138,15 +154,40 @@ public function getId()
     }
 
     /**
-     * @return array
+     * @return string
      * @throws \core_kernel_persistence_Exception
      */
+    public function getProctorBypassed(){
+        if($this->proctored === null){
+            $byPass = $this->getCurrentResource($this->id)->getOnePropertyValue(new core_kernel_classes_Property(EligibilityService::PROPERTY_BYPASSPROCTOR_URI));
+            $this->proctored = $byPass instanceof \core_kernel_classes_Resource
+                ? $byPass->getUri() == EligibilityService::BOOLEAN_TRUE
```

Comment:
If $byPass is an object we have to return `$byPass->getUri()`. Now it returns boolean.